pkg/nvkind/cluster.go

```diff
@@ -123,17 +134,53 @@ func (c *Cluster) Create(opts ...ClusterCreateOption) error {
 	return nil
 }
 
+func (c *Cluster) PrintClusterInfo() error {
+	// A note on `kubectl` invocations: not specifying `--context <name>` would
+	// propbably work in most of the cases because `kind` also sets the default
```

Comment:
```suggestion
	// propably work in most of the cases because `kind` also sets the default
```

Reply:
Thanks. Reminded me that I like and want to use [this spell check extension for vscode](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker).

It finds such things:
![image](https://github.com/user-attachments/assets/b21b91dc-121a-413a-b28a-f711b99fe1ad)

Reply:
done